sdk/network/Azure.ResourceManager.Network/api/Azure.ResourceManager.Network.netstandard2.0.cs

```diff
@@ -8977,13 +10900,49 @@ public partial class ServiceTagsListResult : Azure.ResourceManager.Models.Resour
         public string NextLink { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Network.Models.ServiceTagInformation> Values { get { throw null; } }
     }
+    public partial class SignatureOverridesFilterValuesQuery
+    {
+        public SignatureOverridesFilterValuesQuery() { }
+        public string FilterName { get { throw null; } set { } }
+    }
+    public partial class SignatureOverridesFilterValuesResponse
```

Comment:
rename to `SignatureOverridesFilterValuesResult`

Reply:
Made the change in this commit https://github.com/Azure/azure-sdk-for-net/pull/34959/commits/70d74ef481f2b2243288810f62a0e6e891bef8ae